include/slang/numeric/ConstantValue.h

```diff
@@ -99,7 +99,6 @@ class ConstantValue {
     SVInt& integer() & { return std::get<SVInt>(value); }
     const SVInt& integer() const& { return std::get<SVInt>(value); }
     SVInt integer() && { return std::get<SVInt>(std::move(value)); }
-    SVInt integer() const&& { return std::get<SVInt>(std::move(value)); }
```

Comment:
The point of the rvalue overloads here is to prevent a trap where otherwise calling these on a temporary could lead to a dangling reference. It's true that const rvalues aren't very useful or likely, but people can create them so for symmetry / safety I include these overloads as well. I think the right fix for the lint warning is to just remove the std::move from these cases.

Reply:
I will restore ConstantValue.h to the previous version, and fix Bag.h (std::forward\<T\>).